pom.xml

```diff
@@ -76,6 +78,11 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.msk</groupId>
+            <artifactId>aws-msk-iam-auth</artifactId>
+            <version>${msk.auth.version}</version>
```

Comment:
@davideicardi What's your view on this? I'd rather not become dependent on Amazon JARs. The code doesn't need them, but the dependency exists when ``sasl.client.callback.handler.class`` (``SASL_CLIENT_CALLBACK_HANDLER_CLASS``) is set to ``software.amazon.msk.auth.iam.IAMClientCallbackHandler``.

I propose to resolve this by changing ``kafdrop.sh`` to allow mounting a folder with extra classes. I recently did that for another project, see hapifhir/hapi-fhir-jpaserver-starter#514

Reply:
Yes, it seems to be a good idea to me. Loading extra classes could be a nice feature also for other cases.

We just have to write a good documentation on this kind of stuff.